packages/react-native-toast/src/ReactNativeToastContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react'
+
+import type { ReactNativeToastInfo } from './types/react-native-toast.types'
+
+type IReactNativeToastContext = {
```

Comment:
Why the `IReactNativeToastContext` and not just `ReactNativeToastContext`

Reply:
Leftover from how it was done in MCC, can be changed, I think